packages/core/src/Toast/context.ts

```diff
@@ -7,6 +7,7 @@ export interface SimpleToast {
   readonly message?: string
   readonly onClose?: (id: ToastId) => void
   readonly duration?: number
+  readonly hasCloseButton?: boolean
```

Comment:
If only Progress Toast needs this prop, you can just add it to `ProgressToast` instead of `SimpleToast`.

Reply:
Thanks for the review. :)

Yes , we need it also in `LoadingToast` (and maybe other toasts as well in the future), however only `ProgressToast` needs a `CSS` tweak.

Reply:
Ah, ok. I understand.